tests/test_opSimOrdering.py

```diff
@@ -0,0 +1,76 @@
+import os
```

Comment:
Thanks for adding unit tests for this work!  I'm a little concerned that this test code is essentially doing the same full processing of the opsimdb file that the `get_twinkles_visits.py` executable would do.  When I ran the executable just now, it took 20 mins on a generic rhel6-64 box at slac from the interactive pool, which seems too long for a set of unit tests.  (The unit tests are still cranking away after 12mins).  For testing purposes, I think it would be preferable to have a small opsimdb file that's a tiny subset of the full minion_1016 file.  We could put it in the `data` folder and so not have to worry about setting `self.obsSimDBPath` in a context-dependent way.